packages/slice-machine/src/modules/customTypes/index.ts

```diff
@@ -91,15 +92,32 @@ export const customTypesReducer: Reducer<
 export function* createCustomTypeSaga({
   payload,
 }: ReturnType<typeof createCustomTypeCreator.request>) {
-  const newCustomType = createCustomType(
-    payload.id,
-    payload.label,
-    payload.repeatable
-  );
-  yield call(saveCustomType, newCustomType, {});
-  yield put(createCustomTypeCreator.success({ newCustomType }));
-  yield put(modalCloseCreator({ modalKey: ModalKeysEnum.CREATE_CUSTOM_TYPE }));
-  yield put(push(`/cts/${payload.id}`));
+  try {
+    const newCustomType = createCustomType(
+      payload.id,
+      payload.label,
+      payload.repeatable
+    );
+    yield call(saveCustomType, newCustomType, {});
+    yield put(createCustomTypeCreator.success({ newCustomType }));
+    yield put(
+      modalCloseCreator({ modalKey: ModalKeysEnum.CREATE_CUSTOM_TYPE })
+    );
+    yield put(push(`/cts/${payload.id}`));
+    yield put(
+      openToasterCreator({
+        message: "Custom type saved",
+        type: ToasterType.SUCCESS,
+      })
+    );
+  } catch (e) {
+    yield put(
+      openToasterCreator({
```

Comment:
Question, if Events can be used that way, why do we use dispatch in the frontend usually in the actions.

Or the other way around why don't we need it here ?